packages/components/src/components/modal/component.tsx

```diff
@@ -1,12 +1,12 @@
 /* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
 import type { KoliBriModalEventCallbacks, LabelPropType, ModalAPI, ModalStates } from '@public-ui/schema';
 import { featureHint, setState, validateLabel, watchString, watchValidator } from '@public-ui/schema';
-import { Component, h, Host, Prop, State, Watch } from '@stencil/core';
-
-import { getKoliBri } from '../../utils/dev.utils';
+import { Component, Host, Prop, State, Watch, h } from '@stencil/core';
 
 import type { JSX } from '@stencil/core';
-import type { ModalService } from './service';
+import { ModalService } from './service';
+
+const Modal = new ModalService();
```

Comment:
```suggestion
const modal = new ModalService();
```
Klein da eine Instanz und keine Konstruktor oder Type